.jenkinsci/test-launcher.groovy

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env groovy
+
+// format the enum elements output like "(val1|val2|...|valN)*"
+def printRange(start, end) {
+  def output = ""
+  for (type in start..end) {
+    output = [output, (type.name() != start.toString() ? "|" : ""), type.name()].join('')
+  }
+  return ["(", output, ")*"].join('')
+}
+
+// return tests list regex that will be launched by ctest
+def chooseTestType() {
+	if (params.merge_pr) {
+		if (env.NODE_NAME.contains('x86_64')) {
```

Comment:
same about enums

Reply:
@muratovv My concern is to leave this update for the next PR on CI (not as major as this one)